src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -135,19 +165,56 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('active', SwitchType::class, [
+                'label' => $this->trans('Enabled', 'Admin.Global'),
+                'help' => $this->trans(
+                        'If you want a category to appear in your store\'s menu, configure your menu module in [1]Modules > Module Manager[/1].',
+                        'Admin.Catalog.Help',
+                        [
+                            '[1]' => '<a href="' . $this->router->generate('admin_module_manage') . '" target="_blank" rel="noopener noreferrer nofollow">',
+                            '[/1]' => '</a>',
+                        ]
+                    ),
                 'required' => false,
             ])
-            ->add('cover_image', FileType::class, [
+            ->add('cover_image', ImageWithPreviewType::class, [
+                'label' => $this->trans('Category cover image', 'Admin.Catalog.Feature'),
+                'help' => $this->trans('This is the cover image for your category: it will be displayed on the category\'s page. The description will appear in its top-left corner.', 'Admin.Catalog.Help'),
                 'required' => false,
+                'can_be_deleted' => true,
+                'show_size' => true,
+                'csrf_delete_token' => 'delete-cover-image',
```

Comment:
```suggestion
                'csrf_delete_token_id' => 'delete-cover-image',
```
Otherwise it's confusing, you don't specify the token itself here but the ID/seed/key used to generate it in the twig file

Reply:
Good point, changed it to id